testing/references/bestest_hydronic/plot_references.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python2
```

Comment:
I think it'd be better to add this kind of functionality in the `testing/utilities.py` script and to remove this one from this PR. I have a script that does exactly this and is general for all test cases. I'll raise an issue for this.

Reply:
Thanks for the catch.  Yes, didn't mean to include.

Reply:
done in https://github.com/ibpsa/project1-boptest/pull/314/commits/d8851fb75af5911cfdf2d089e8235238b2f71ac0.